staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -75,14 +79,14 @@ type Event struct {
 	// API object from the request, in JSON format. The RequestObject is recorded as-is in the request
 	// (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or
 	// merging. It is an external versioned object type, and may not be a valid object on its own.
-	// Omitted for non-resource requests.  Only logged at RequestObject Level and higher.
+	// Omitted for non-resource requests.  Only logged at Request Level and higher.
 	// +optional
-	RequestBody string
+	RequestObject runtime.Unknown
```

Comment:
> How does this help with filters? Filters will not be done by the backend, but before writing it into the event.

I'd like the object shape to be inspectable if the sync does something like write it directly to a different etcd. Not nice, not good, but having the object directly embedded means that anyone looking at an `Event.audit.k8s.io` can run jsonpath and jq sanely against the object itself. I don't think that stringifying json inside of (likely) json is path we'd be happy with.  We encouraged servicecatalog to avoid it for much the same reason.